### ISSUE TITLE
Unknown Issue -- Perimeter Reachability spikes after several iterations

### ISSUE THREAD

**Issue by ifndefJOSH** [fluentverification/stamina-storm#9]
Output (Annotated):
```
=====================================================================================
Data is correct here: (checked against the Java version)
=====================================================================================
[INFO]: Starting STAMINA with kappa = 1.000000 and reduction factor = 1.250000
[INFO]: Stamina version is: 0.1
[INFO]: There are the following number of state labels: 0
P=? [true U[0, 2100] ((LacI < 20) & (TetR > 40))]
[WARNING]: This property (1) is a probability path formula.
[INFO]: Approximation [Refine Iterations: 0, kappa = 1.000000]
piHat = 1 and w/approx = 0.0005
[INFO]: Perimeter reachability: 1.000000
At this iteration, kappa = 1
[INFO]: Finished state space truncation. Explored 4 states in total.
At this iteration, the following states are terminal:3
piHat = 3 and w/approx = 0.0005
[INFO]: Perimeter reachability: 3.000000
At this iteration, kappa = 0.8
[INFO]: Finished state space truncation. Explored 10 states in total.
At this iteration, the following states are terminal:7
piHat = 5.6 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.600000
At this iteration, kappa = 0.64
[INFO]: Finished state space truncation. Explored 16 states in total.
At this iteration, the following states are terminal:11
piHat = 7.04 and w/approx = 0.0005
[INFO]: Perimeter reachability: 7.040000
At this iteration, kappa = 0.512
[INFO]: Finished state space truncation. Explored 22 states in total.
At this iteration, the following states are terminal:15
piHat = 7.68 and w/approx = 0.0005
[INFO]: Perimeter reachability: 7.680000
At this iteration, kappa = 0.4096
[INFO]: Finished state space truncation. Explored 28 states in total.
At this iteration, the following states are terminal:19
piHat = 7.7824 and w/approx = 0.0005
[INFO]: Perimeter reachability: 7.782400
At this iteration, kappa = 0.32768
[INFO]: Finished state space truncation. Explored 31 states in total.
At this iteration, the following states are terminal:21
piHat = 6.88128 and w/approx = 0.0005
[INFO]: Perimeter reachability: 6.881280
At this iteration, kappa = 0.262144
[INFO]: Finished state space truncation. Explored 35 states in total.
At this iteration, the following states are terminal:23
piHat = 6.02931 and w/approx = 0.0005
[INFO]: Perimeter reachability: 6.029312
At this iteration, kappa = 0.209715
[INFO]: Finished state space truncation. Explored 39 states in total.
At this iteration, the following states are terminal:25
piHat = 5.24288 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.242880
At this iteration, kappa = 0.167772
[INFO]: Finished state space truncation. Explored 45 states in total.
At this iteration, the following states are terminal:28
piHat = 4.69762 and w/approx = 0.0005
[INFO]: Perimeter reachability: 4.697620
At this iteration, kappa = 0.134218
[INFO]: Finished state space truncation. Explored 49 states in total.
At this iteration, the following states are terminal:30
piHat = 4.02653 and w/approx = 0.0005
[INFO]: Perimeter reachability: 4.026532
At this iteration, kappa = 0.107374
[INFO]: Finished state space truncation. Explored 53 states in total.
At this iteration, the following states are terminal:32
piHat = 3.43597 and w/approx = 0.0005
[INFO]: Perimeter reachability: 3.435974
=====================================================================================
This is where the data becomes incorrect below this iteration
=====================================================================================
At this iteration, kappa = 0.0858993
[INFO]: Finished state space truncation. Explored 119 states in total.
At this iteration, the following states are terminal:69
piHat = 5.92705 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.927055
At this iteration, kappa = 0.0687195
[INFO]: Finished state space truncation. Explored 181 states in total.
At this iteration, the following states are terminal:114
piHat = 7.83402 and w/approx = 0.0005
[INFO]: Perimeter reachability: 7.834020
At this iteration, kappa = 0.0549756
[INFO]: Finished state space truncation. Explored 210 states in total.
At this iteration, the following states are terminal:130
piHat = 7.14683 and w/approx = 0.0005
[INFO]: Perimeter reachability: 7.146826
At this iteration, kappa = 0.0439805
[INFO]: Finished state space truncation. Explored 237 states in total.
At this iteration, the following states are terminal:147
piHat = 6.46513 and w/approx = 0.0005
[INFO]: Perimeter reachability: 6.465128
At this iteration, kappa = 0.0351844
[INFO]: Finished state space truncation. Explored 288 states in total.
At this iteration, the following states are terminal:183
piHat = 6.43874 and w/approx = 0.0005
[INFO]: Perimeter reachability: 6.438740
At this iteration, kappa = 0.0281475
[INFO]: Finished state space truncation. Explored 338 states in total.
At this iteration, the following states are terminal:217
piHat = 6.10801 and w/approx = 0.0005
[INFO]: Perimeter reachability: 6.108007
At this iteration, kappa = 0.022518
[INFO]: Finished state space truncation. Explored 383 states in total.
At this iteration, the following states are terminal:240
piHat = 5.40432 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.404320
At this iteration, kappa = 0.0180144
[INFO]: Finished state space truncation. Explored 441 states in total.
At this iteration, the following states are terminal:269
piHat = 4.84587 and w/approx = 0.0005
[INFO]: Perimeter reachability: 4.845873
At this iteration, kappa = 0.0144115
[INFO]: Finished state space truncation. Explored 589 states in total.
At this iteration, the following states are terminal:357
piHat = 5.14491 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.144912
At this iteration, kappa = 0.0115292
[INFO]: Finished state space truncation. Explored 709 states in total.
At this iteration, the following states are terminal:430
piHat = 4.95756 and w/approx = 0.0005
[INFO]: Perimeter reachability: 4.957562
At this iteration, kappa = 0.00922337
[INFO]: Finished state space truncation. Explored 923 states in total.
At this iteration, the following states are terminal:550
piHat = 5.07285 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.072855
At this iteration, kappa = 0.0073787
[INFO]: Finished state space truncation. Explored 1183 states in total.
At this iteration, the following states are terminal:683
piHat = 5.03965 and w/approx = 0.0005
[INFO]: Perimeter reachability: 5.039650
At this iteration, kappa = 0.00590296
[INFO]: Finished state space truncation. Explored 1446 states in total.
At this iteration, the following states are terminal:792
piHat = 4.67514 and w/approx = 0.0005
[INFO]: Perimeter reachability: 4.675143
At this iteration, kappa = 0.00472237
```

**Comment by ifndefJOSH** [comment]
reference commit #e47b1e9b02ae60dd1772bf4b7f61af609c53e21f

**Comment by ifndefJOSH** [comment]
Fixed with commit 2f8a0b113590a1a302586661f64ee35e6489f2f4
